internal/codegen/stencil.go

```diff
@@ -67,73 +64,20 @@ type Stencil struct {
 	lock *stencil.Lockfile
 
 	// modules is a list of modules used in this stencil render
-	modules      []*modules.Module
-	moduleCaller *ModuleCaller
+	modules []*modules.Module
 
-	// isFirstPass denotes if the renderer is currently in first
-	// pass mode
-	isFirstPass bool
+	// isRenderStage denotes if we're in the initial render stage. This is
```

Comment:
I'd say "pre-render stage", this is confusing as worded now

Reply:
I was initially thinking about making this something like "RenderStage" and having it be either `PreRender" or "FinalRender". Thoughts?

Reply:
i'm just talking about the comment.  "isRenderStage" is fine to me.

Reply:
o well, i switched it to a type which means we could add more stages later 🤷